site/src/pages/DeploySettingsPage/NotificationsPage/NotificationEvents.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { spyOn, userEvent, within } from "@storybook/test";
+import { API } from "api/api";
+import { selectTemplatesByGroup } from "api/queries/notifications";
+import type { DeploymentValues } from "api/typesGenerated";
+import { MockNotificationTemplates } from "testHelpers/entities";
+import { NotificationEvents } from "./NotificationEvents";
+import { baseMeta } from "./storybookUtils";
+
+const meta: Meta<typeof NotificationEvents> = {
+	title: "pages/DeploymentSettings/NotificationsPage/NotificationEvents",
+	component: NotificationEvents,
+	args: {
+		defaultMethod: "smtp",
+		availableMethods: ["smtp", "webhook"],
+		templatesByGroup: selectTemplatesByGroup(MockNotificationTemplates),
+		deploymentValues: baseMeta.parameters.deploymentValues,
+	},
+	...baseMeta,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof NotificationEvents>;
+
+export const NoEmailSmarthost: Story = {
+	args: {
+		deploymentValues: {
+			notifications: {
+				webhook: {
+					endpoint: "https://example.com",
+				},
+				email: {
+					smarthost: "",
```

Comment:
There are 3 required fields for SMTP: https://coder.com/docs/admin/notifications#smtp-email

Reply:
Related to what I commented here https://github.com/coder/coder/pull/14565#discussion_r1745360260